### ISSUE TITLE
Use Tab key to validate input change on Slider

### ISSUE THREAD

**Issue by chaffra** [jupyter-widgets/ipywidgets#2535]
I noticed the `Tab` does not change the value on a FloatSlider for example. To reproduce, type a new value in the attached text box, press `Tab' instead of `Enter`. You move to a different focus but the slider does not get update. 
* Is this the intended behavior? 
* Should a value change be triggered when the widget looses focus and its text entry actually changed?

**Comment by jkochNU** [comment]
I'd like to echo the above questions. The current behavior can lead to bizarre situations: the text entry associated with the slider shows one value, while the slider is actually at a different value internally.  I would go as far as saying that this cannot be intended behavior. 

I am surprised nobody has responded to this for 3 years - am I overlooking something?

**Comment by i3v** [comment]
I strongly agree this should be fixed somehow. As for me, this is one of the most annoying issues.
